app/controllers/citizens/property_values_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Citizens
+  class PropertyValuesController < BaseController
+    def show
+      @form = Applicants::PropertyValueForm.new # this is so errors are visible
+    end
+
+    def create
+      @form = Applicants::PropertyValueForm.new(edit_params)
+
+      if @form.save
+        render plain: 'Navigate to question 1c; What is the outstanding mortgage?'
+      else
+        render :show
+      end
+    end
+
+    private
+
+    def property_value_params
+      params.fetch(:legal_aid_application, {}).permit(:property_value)
```

Comment:
The correct syntax for handling params is:
```ruby
params.require(:legal_aid_application).permit(:property_value)
```
See: https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters